group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -2904,6 +2908,135 @@ private void validateGenericGroupHeartbeat(
         }
     }
 
+    /**
+     * Handle a generic group LeaveGroup request.
```

Comment:
shouldn't generic Leave Group request suffice? instead of generic group leave group